--- origsrc/phonon-4.3.0/gstreamer/CMakeLists.txt	2008-11-24 11:12:02.000000000 -0600
+++ src/phonon-4.3.0/gstreamer/CMakeLists.txt	2009-01-26 14:54:10.844250000 -0600
@@ -56,7 +56,7 @@ if (BUILD_PHONON_GSTREAMER)
          )
    endif(ALSA_FOUND AND NOT GSTREAMER_HAS_NONBLOCKING_ALSASINK)
 
-   automoc4_add_library(phonon_gstreamer SHARED ${phonon_gstreamer_SRCS})
+   automoc4_add_library(phonon_gstreamer MODULE ${phonon_gstreamer_SRCS})
    set_target_properties(phonon_gstreamer PROPERTIES PREFIX "")
    target_link_libraries(phonon_gstreamer
       ${QT_QTOPENGL_LIBRARY}
--- origsrc/phonon-4.3.0/xine/CMakeLists.txt	2009-01-08 08:47:18.000000000 -0600
+++ src/phonon-4.3.0/xine/CMakeLists.txt	2009-01-26 14:53:51.781750000 -0600
@@ -102,7 +102,7 @@ if(XINE_FOUND)
     configure_file(config-xine-widget.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-xine-widget.h )
     
     add_definitions(-DPHONON_BACKEND_VERSION_4_2)
-    automoc4_add_library(phonon_xine SHARED ${phonon_xine_SRCS})
+    automoc4_add_library(phonon_xine MODULE ${phonon_xine_SRCS})
     set_target_properties(phonon_xine PROPERTIES PREFIX "")
     target_link_libraries(phonon_xine ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${PHONON_LIBS} ${XINE_LIBRARY})
     if(XCB_FOUND AND XINE_XCB_FOUND)
