--- origsrc/phonon-4.4.4/cmake/FindPhononInternal.cmake	2011-01-19 13:20:00.000000000 -0600
+++ src/phonon-4.4.4/cmake/FindPhononInternal.cmake	2011-02-01 22:09:12.107443700 -0600
@@ -201,7 +201,7 @@ else (WIN32)
     _set_fancy(EXEC_INSTALL_PREFIX     "${CMAKE_INSTALL_PREFIX}"                   "Base directory for executables and libraries")
 endif (WIN32)
 _set_fancy(SHARE_INSTALL_PREFIX        "${EXEC_INSTALL_PREFIX}/share"              "Base directory for files which go to share/")
-_set_fancy(INCLUDE_INSTALL_DIR         "${EXEC_INSTALL_PREFIX}/include"            "The subdirectory to the header prefix")
+_set_fancy(INCLUDE_INSTALL_DIR         "${EXEC_INSTALL_PREFIX}/include/kde4"       "The subdirectory to the header prefix")
 _set_fancy(BIN_INSTALL_DIR             "${EXEC_INSTALL_PREFIX}/bin"                "The install dir for executables (default ${EXEC_INSTALL_PREFIX}/bin)")
 _set_fancy(LIB_INSTALL_DIR             "${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX}"   "The subdirectory relative to the install prefix where libraries will be installed (default is ${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX})")
 _set_fancy(PLUGIN_INSTALL_DIR          "${LIB_INSTALL_DIR}/kde4"                   "The subdirectory relative to the install prefix where plugins will be installed (default is ${LIB_INSTALL_DIR}/kde4)")
